mcs/class/corlib/Test/System.IO/DriveInfoTest.cs

```diff
@@ -54,7 +54,22 @@ public void Constructor ()
 		[Test]
 		public void ConstructorThrowsOnNonExistingDrive ()
 		{
-			Assert.Throws<ArgumentException> (() => new DriveInfo ("/monodriveinfotest"));
+			Assert.Throws<ArgumentException> (() => new DriveInfo ("monodriveinfotest"));
+		}
+
+		[Test]
+		[Category ("NotWasm")] // it doesn't know about 'memfs' drive format
+		public void ConstructorGetsValidDriveFromNonDriveString ()
+		{
+			if (Environment.OSVersion.Platform != PlatformID.Win32NT && Environment.OSVersion.Platform != PlatformID.MacOSX)
+				Assert.Ignore ("Some Linux-hosted CI builders don't have '/' mounted, just testing Windows and MacOS for now.");			
```

Comment:
@MaximLipnin `Environment.OSVersion.Platform` for historical reasons returns `Unix` on macOS so right now this only runs on Windows.

Reply:
@akoeplinger Thanks for info. I tested it on my local macOS env so I assume it should pass on CI too in case of switching to "right" macOS platform id.

Reply:
@MaximLipnin yeah if you want to enable it on macOS I'd switch to the newer RuntimeInformation APIs:

```
if (!RuntimeInformation.IsOSPlatform (OSPlatform.Windows) && !RuntimeInformation.IsOSPlatform (OSPlatform.OSX))
  Assert.Ignore ("Some Linux-hosted CI builders don't have '/' mounted, just testing Windows and MacOS for now.");
```

Reply:
Ok, I'll make this change.

Reply:
Updated in https://github.com/mono/mono/pull/13205